Classes/Notifications/NotificationClient.swift

```diff
@@ -69,14 +81,61 @@ final class NotificationClient {
         ) { response, nextPage in
             if let notifications = (response.value as? [[String:Any]])?.flatMap({ NotificationResponse(json: $0) }) {
                 let viewModels = CreateViewModels(containerWidth: width, notifications: notifications)
-                cache.set(values: viewModels)
-                completion(.success((viewModels, nextPage?.next)))
+                self.fetchStates(for: viewModels, page: nextPage?.next, completion: completion)
             } else {
                 completion(.error(response.error))
             }
         })
     }
 
+    private func fetchStates(
+        for notifications: [NotificationViewModel],
+        page: Int?,
+        completion: @escaping (Result<([NotificationViewModel], Int?)>) -> Void
+        ) {
+
+        let content = "state comments{totalCount}"
+        let notificationQueries: String = notifications.flatMap {
+            guard let alias = $0.stateAlias else { return nil }
+            return """
+            \(alias.key): repository(owner: "\($0.owner)", name: "\($0.repo)") { issueOrPullRequest(number: \(alias.number)) { ...on Issue {\(content)} ...on PullRequest {\(content)} } }
```

Comment:
Any reason why you chose to put this in a multiline comment, but then put it all on one line anyway?
Would prefer it if this/the content var to be actual multilines so it's easier to read!

Reply:
Escaped quotes looked gross, and GQL has special requirements for newlines. Plus this was a hack that soon as it was working I shipped 😄

Feel free to PR with better formatting!